plugins/catalog-backend-module-gitlab/src/lib/types.ts

```diff
@@ -70,6 +80,31 @@ export type GitLabGroupMembersResponse = {
   };
 };
 
+export type GitLabSaasGroupsResponse = {
```

Comment:
Maybe we get rid of the "saas" naming here too?
```suggestion
export type GitLabDescendantGroupsResponse = {
```

Reply:
Done